plugin/fzf.vim

```diff
@@ -342,7 +342,8 @@ function! s:common_sink(action, lines) abort
 endfunction
 
 function! s:get_color(attr, ...)
-  let gui = !s:is_win && !has('win32unix') && has('termguicolors') && &termguicolors
+  " Force 24 bit colors: g:fzf_force_24_bit_colors
+  let gui = (exists('g:fzf_force_24_bit_colors') && g:fzf_force_24_bit_colors) || (!s:is_win && !has('win32unix') && has('termguicolors') && &termguicolors)
```

Comment:
I would reuse the name of an existing Vim option, rather than coin a new name.

```suggestion
  " Force 24 bit colors: g:fzf_force_termguicolors (temporary workaround for https://github.com/junegunn/fzf.vim/issues/1152)
  let gui = get(g:, 'fzf_force_termguicolors', 0) || (!s:is_win && !has('win32unix') && has('termguicolors') && &termguicolors)
```